src/workerd/server/tests/python/BUILD.bazel

```diff
@@ -30,6 +30,8 @@ py_wd_test("subdirectory")
 
 py_wd_test(
     "sdk",
+    # TODO: will need to rebuild bundle to change the `status_text` field name
```

Comment:
Did you do this?

Reply:
I was planning on merging this in first, but thinking about it again I suppose I can rebuild the bundle from this branch and update it in this PR